VideoWeb/VideoWeb/ClientApp/src/app/services/models/instant-message.ts

```diff
@@ -2,10 +2,12 @@ import { ChatResponse, IChatResponse } from '../clients/api-client';
 
 export interface IInstantMessage extends IChatResponse {
     conferenceId?: string;
+    isJudge?: boolean;
```

Comment:
No neeed to extend the model here, not a suitable solution for when participant IM is enabled.

Reply:
done